events/equeue/equeue.c

```diff
@@ -245,20 +256,21 @@ static int equeue_enqueue(equeue_t *q, struct equeue_event *e, unsigned tick) {
 
     // notify background timer
     if ((q->background.update && q->background.active) &&
-        (q->queue == e && !e->sibling)) {
+            (q->queue == e && !e->sibling)) {
         q->background.update(q->background.timer,
-                equeue_clampdiff(e->target, tick));
+                             equeue_clampdiff(e->target, tick));
```

Comment:
What rule is in effect here? 3 tabs after newlin in an expression? Why are if statements different?

Reply:
Then what's going on with this line? https://github.com/ARMmbed/mbed-os/pull/6014#discussion-diff-167327409R303

This seems like it will push lines really far off the page, especially with the longer namespaced names in C++.